api/v1alpha1/etcd_types.go

```diff
@@ -393,3 +393,92 @@ type EtcdList struct {
 	Items           []Etcd `json:"items"`
 }
+
+// LastOperationState is a string alias.
+type LastOperationState string
+
+const (
+	// LastOperationStateActive indicates that an operation is ongoing.
+	LastOperationStateActive LastOperationState = "Active"
+	// LastOperationStateSucceeded indicates that an operation has completed successfully.
+	LastOperationStateSucceeded LastOperationState = "Succeeded"
+	// LastOperationStateFailed indicates that an operation is completed with errors and won't be retried.
+	LastOperationStateFailed LastOperationState = "Failed"
+)
+
+// LastOperation indicates the type and the state of the last operation, along with a description
+// message and a progress indicator.
+type LastOperation struct {
+	// Description is a human readable message indicating details about the last operation.
+	Description string `json:"description" protobuf:"bytes,1,opt,name=description"`
+	// LastUpdateTime is the last time the operation state transitioned from one to another.
+	LastUpdateTime metav1.Time `json:"lastUpdateTime" protobuf:"bytes,2,opt,name=lastUpdateTime"`
+	// State is the status of the last operation, one of Aborted, Processing, Succeeded, Error, Failed.
+	State LastOperationState `json:"state" protobuf:"bytes,4,opt,name=state,casttype=LastOperationState"`
+}
+
+// EtcdCopyBackupsTaskSpec defines the parameters for the copy operation
+type EtcdCopyBackupsTaskSpec struct {
+	// SourceStore defines the specification of object store provider for storing backups.
+	// +required
+	SourceStore *StoreSpec `json:"sourceStore,omitempty"`
+	// TargetStore defines the specification of object store provider for storing backups.
+	// +required
+	TargetStore *StoreSpec `json:"targetStore,omitempty"`
+	// MaxBackupAge is the maximum age in days that a snapshot must have in order to be copied.
+	// By default all snapshots will be copied.
+	// +optional
+	MaxBackupAge *uint32 `json:"maxBackupAge,omitempty"`
+	// MaxBackups is the maximum number of snapshots that will be copied starting with the most recent ones.
+	// +optional
+	MaxBackups *uint32 `json:"maxBackups,omitempty"`
+	// Image defines the backup-restore container image and tag
+	// +optional
+	Image *string `json:"image,omitempty"`
+}
+
+// EtcdCopyBackupsTaskStatus defines parameters that contains the current state of the etcd backup copy operation
+type EtcdCopyBackupsTaskStatus struct {
+	// ObservedGeneration is the most recent generation observed for this resource.
+	// +optional
+	ObservedGeneration *int64 `json:"observedGeneration,omitempty"`
+	// LastOperation represents the latest available observations of an etcd backup copy task current state.
+	// +optional
+	LastOperation *LastOperation `json:"lastOperation,omitempty"`
+	// TaskReference represents the details of the task that is currently deployed to preform the copy operation.
+	// +optional
+	ObjectRef *corev1.TypedLocalObjectReference `json:"objectRef,omitempty"`
```

Comment:
```suggestion
	TaskReference *corev1.TypedLocalObjectReference `json:"taskReference,omitempty"`
```

Reply:
I rather addressed it because the doc string does not match the renamed field. Can you change the doc string then?